.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!--
+IMPORTANT: Please use the following link to create a new issue:
+
+https://nuxtjs.cmty.io/issues/new
```

Comment:
I don't think we can use CMTY for that module at the moment

Reply:
@manniL 
I thought I would have a channel for each module
Should I remove this file?

Reply:
Removed!